webapp/publisher/snaps/views.py

```diff
@@ -1117,14 +1117,16 @@ def get_publicise(snap_name):
     available_languages = {
         "en": {"title": "English", "text": "Get it from the Snap Store"},
         "de": {"title": "Deutsch", "text": "Installieren vom Snap Store"},
-        "es": {"title": "Español", "text": "Instalar desde Snap Store"},
+        "jp": {"title": "日本語", "text": "Snap Store から入手ください"},
         "fr": {
             "title": "Français",
             "text": "Installer à partir du Snap Store",
         },
-        "jp": {"title": "日本語", "text": "Snap Store から入手ください"},
-        "ru": {"title": "русский язык", "text": "Загрузите из Snap Store"},
+        "es": {"title": "Español", "text": "Instalar desde Snap Store"},
         "tw": {"title": "中文（台灣）", "text": "安裝軟體敬請移駕 Snap Store"},
+        "ru": {"title": "русский язык", "text": "Загрузите из Snap Store"},
+        "pt": {"title": "Português", "text": "Disponível na Snap Store"},
+        "it": {"title": "Italiano", "text": "Scarica dallo Snap Store"},
```

Comment:
Are these in any particular order?
I think they used to be alphabetical (with English as default on top), now order shifted and seems random.